translationCoreApps/translationNotes_Check_plugin
# Backtrack sources of Ephesiasn

#### This pull request addresses:

This pull request addresses #857 by using an older version of Ephesians. 


#### How to test this pull request:

Take a look at the spreadsheet on #857, and make sure Epehsians and Titus have the appropriate amount of checks.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/translationnotes_check_plugin/64)
